### ISSUE TITLE
ImagePullSecrets support

### ISSUE THREAD

**Issue by extravio** [sonatype/nexus-iq-server-ha#34]
We have policies in place to enforce that images are pulled from our private Docker repository.

We need the chart to support `ImagePullSecrets`.

**Comment by ehuizar1028** [comment]
yes, we need this too, please add support for this, it's easy to do.
